.github/actions/semantic-pr-footer-v1/src/isValidFooter.ts

```diff
@@ -1,12 +1,26 @@
+const validFooters = [
+  // github terms that link an issue
+  // @see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
+  'close: ',
```

Comment:
It appears GitHub closes the referenced issue with `:` omitted, should we remove the use of `:`? 

example: https://github.com/dequelabs/zidious-testing/pull/125

Reply:
I don't think so. It's true that it can close without the `:`, but that leads to odd situations where you write a paragraph `"I thought about this to close #40"` and github automatically links the ticket to close. For the footer I'd rather be explicit and put the `:` otherwise you could get away with a footer that is invalid but passes like `"closes the loophole"`